packages/transmute-cli/src/commands/provision.js

```diff
@@ -1,46 +1,52 @@
 const run = require('./runner');
-const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION ||  'v1.9.0';
-const MINIKUBE_CPU = process.env.MINIKUBE_CPU ||  '4';
-const MINIKUBE_MEMORY = process.env.MINIKUBE_MEMORY ||  '4096';
-const MINIKUBE_DISK = process.env.MINIKUBE_DISK ||  '100g';
-const MINIKUBE_PROFILE = process.env.MINIKUBE_PROFILE ||  'transmute-k8s';
+const TRANSMUTE_KUBE_VERSION = process.env.TRANSMUTE_KUBE_VERSION || 'v1.9.0';
+const MINIKUBE_CPU = process.env.MINIKUBE_CPU || '4';
+const MINIKUBE_MEMORY = process.env.MINIKUBE_MEMORY || '4096';
+const MINIKUBE_DISK = process.env.MINIKUBE_DISK || '100g';
+const MINIKUBE_PROFILE = process.env.MINIKUBE_PROFILE || 'transmute-k8s';
 
-export function minikube( dryrun, clusterName, minikubeDriver ) {
-  const minikube_start = 'minikube start '
-    + ' --kubernetes-version ' +  TRANSMUTE_KUBE_VERSION
-    + ' --disk-size ' +  MINIKUBE_DISK
-    + ' --cpus ' +  MINIKUBE_CPU
-    + ' --memory ' +  MINIKUBE_MEMORY;
-    + ' --profile ' +  MINIKUBE_PROFILE;
+module.exports.minikube = (dryrun, clusterName, minikubeDriver) => {
+  const minikube_start =
+    'sudo minikube start ' +
+    ' --kubernetes-version ' +
+    TRANSMUTE_KUBE_VERSION +
+    ' --disk-size ' +
+    MINIKUBE_DISK +
+    ' --cpus ' +
+    MINIKUBE_CPU +
+    ' --memory ' +
+    MINIKUBE_MEMORY;
+  +' --profile ' + MINIKUBE_PROFILE;
   let prov_cmd = minikube_start + ' --vm-driver=virtualbox';
-  if (minikubeDriver == undefined ) {
+  if (minikubeDriver == undefined) {
     prov_cmd = minikube_start + ' --vm-driver=virtualbox';
-  }
-  else if (minikubeDriver == 'virtualbox' ) {
+  } else if (minikubeDriver == 'virtualbox') {
     prov_cmd = minikube_start + ' --vm-driver=virtualbox';
-  }
-  else if (minikubeDriver == 'kvm' ) {
+  } else if (minikubeDriver == 'kvm') {
     prov_cmd = minikube_start + ' --vm-driver=kvm';
-  }
-  else if (minikubeDriver == 'kvm2' ) {
+  } else if (minikubeDriver == 'kvm2') {
     prov_cmd = minikube_start + ' --vm-driver=kvm2';
-  }
-  else if (minikubeDriver == 'none' ) {
+  } else if (minikubeDriver == 'none') {
     prov_cmd = minikube_start + ' --vm-driver=none';
-  }
-  else {
+  } else {
     prov_cmd = minikube_start + ' --vm-driver=virtualbox';
   }
-  console.log( prov_cmd );
-  if (dryrun === 'true' ) {
-    console.log( '<--dry run-->' );
-  }
-  else {
-    run.ner( prov_cmd );
+  console.log(prov_cmd);
```

Comment:
remove.

Reply:
no, its important to see the command that is running.

Reply:
this was removed